test/e2e/wait_util.go

```diff
@@ -192,3 +193,20 @@ func WaitForDeployment(t *testing.T, kubeclient kubernetes.Interface, namespace,
 	logrus.Infof("Deployment of %s in namespace %s took %s\n", name, namespace, elapsed)
 	return err
 }
+
+// WaitForSecret waits for a secret to be available
+func WaitForSecret(secretName string) {
+	err := wait.Poll(retryInterval, timeout, func() (done bool, err error) {
+		secretList, err := fw.KubeClient.CoreV1().Secrets(kafkaNamespace).List(metav1.ListOptions{IncludeUninitialized: false})
```

Comment:
`kafkaNamespace`? Is it holding the namespace for this test, or is it really the `kafkaNamespace`?

Reply:
Also, I think you can use a `Get` instead of a `List`, no? You know the object's name after all...